compoments/report-repair/address.js

```diff
@@ -56,15 +64,16 @@ const Address = ({handleChange, values, goToStep}) => {
   }
 
   return <div className="govuk-grid-row" data-cy="address">
-    <ComponentHeader title={title} />
     <div className="govuk-grid-column-two-thirds">
       {state.error.msg && <ErrorSummary active={state.activeError} errorSummaryTextAndLocation={[{text:state.error.msg, location: '#address'}]} pageTitle={pageTitle} />}
       <h1 className="govuk-heading-l">{title}</h1>
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-2"> Postcode </h2>
+      <p class="govuk-body"> 
+        <PostcodeChange goToStep={goToStep} postcode={values.postcode} />       
+      </p>
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-2"> {addressPrompt} </h2>
```

Comment:
Are we allowed to use `govuk-!-margin-bottom-2`? I might be wrong but I think I remember we were removing the code with ! in CSS classes.